### PR TITLE
Custom video view and table view cell

### PR DESCRIPTION
**Pull request by collection-man-globerskin**

### CODE REVIEW

File: VideoStreaming/SubViews/Views/VideoView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//  VideoStreaming
+//
+//  Created by Julian Llorensi on 17/09/2019.
+//  Copyright © 2019 ESPN. All rights reserved.
+//
+
+import UIKit
+
+class VideoView: UIView {
+    @IBOutlet weak var videoImageView: UIImageView!
```
Comment (ja-gutierrez) [inline VideoStreaming/SubViews/Views/VideoView.swift]: Please add private to these properties. They should not be able to be set from outside this class.

File: VideoStreaming/SubViews/Views/VideoView.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        setViewInfoWith(video)
+        setUpImageView()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func resetViewInfoWith(_ video: Video) {
+        videoImageView.image = UIImage(named: video.imageName)
```
Comment (ja-gutierrez) [inline VideoStreaming/SubViews/Views/VideoView.swift]: If you see, these two methods are the same (DRY: Don`t Repeat Yourself principle). I think having only one public method is enough. Also, the `With` at the end of method name looks no swifty, ;). You can take a look at this https://swift.org/documentation/api-design-guidelines/ .
I would prefer something like `configure(with video: Video)`

File: VideoStreaming/SubViews/Cells/VideoTableViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
+//
+//  VideoTableViewCell.swift
+//  VideoStreaming
+//
+//  Created by Julian Llorensi on 17/09/2019.
+//  Copyright © 2019 ESPN. All rights reserved.
+//
+
+import UIKit
```
Comment (ja-gutierrez) [inline VideoStreaming/SubViews/Cells/VideoTableViewCell.swift]: There was a ticket for this work, remember that a PR should only have what is expected for the ticket itself. ;).

**Review by ja-gutierrez — CHANGES_REQUESTED** [review]

File: VideoStreaming/SubViews/Views/VideoView.swift (hunk 1 above)
Comment (collection-man-globerskin) [reply VideoStreaming/SubViews/Views/VideoView.swift]: Done

File: VideoStreaming/SubViews/Views/VideoView.swift (hunk 2 above)
Comment (collection-man-globerskin) [reply VideoStreaming/SubViews/Views/VideoView.swift]: Done

**Comment by collection-man-globerskin** [comment]
Screen Shot 2019-09-19 at 13 21 19

**Comment by collection-man-globerskin** [comment]
Screen Shot 2019-09-19 at 14 59 55

**Comment by collection-man-globerskin** [comment]
Screen Shot 2019-09-19 at 15 32 04

**Comment by castadev85** [comment]
https://github.com/dtci-ios/dtci-training-ios/pull/3/files

**Pull request closed without merging**